Linux/parameters.json

```diff
@@ -71,8 +59,17 @@
         "nt0reverseProxyEndpointPort": {
             "value": 19081
         },
+        "adminUserName": {
+            "value": "admin"
```

Comment:
Previous username was `adminuser`, unless there is some reason to change it I would maintain the same value.

Reply:
On Windows it was `dekapur` (I am guessing that comes from the name of the original creator of the template) so I went for `admin` on both Windows and Linux. We can keep the original names if you think it is better, just wanted to have a neutral username on both templates.